### PR TITLE
2022-03-28-spellcheck_dl_en_3_0

### PR DESCRIPTION
**Pull request by jsl-models**

### CODE REVIEW

File: docs/_posts/murat-gunay/2022-03-28-spellcheck_dl_en_3_0.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    .setSuffixes(Array("\"", "”", ".", ",", "?", ")", "]", "!", ";", ":", "'s", "’s"))
+
+  val spellChecker = ContextSpellCheckerModel.
+    pretrained("spellcheck_dl", "en").
+    setInputCols("token").
+    setOutputCol("checked")
+
+val pipeline =  new Pipeline().setStages(Array(assembler, tokenizer, spellChecker))
+val empty_df = spark.createDataFrame([[""]]).toDF("text")
+val lp = LightPipeline(pipeline.fit(empty_df))
```
Comment (josejuanmartinez) [inline docs/_posts/murat-gunay/2022-03-28-spellcheck_dl_en_3_0.md:78]: You need `new` here, you are creating a new object.

File: docs/_posts/murat-gunay/2022-03-28-spellcheck_dl_en_3_0.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+val text = ["During the summer we have the best ueather.", "I have a black ueather jacket, so nice."]
```
Comment (josejuanmartinez) [inline docs/_posts/murat-gunay/2022-03-28-spellcheck_dl_en_3_0.md:79]: Is this array in Scala correct, with brackets? Isn't it Array()?

**Review by josejuanmartinez — CHANGES_REQUESTED** [review]

File: docs/_posts/murat-gunay/2022-03-28-spellcheck_dl_en_3_0.md (hunk 2 above)
Comment (murat-gunay) [reply docs/_posts/murat-gunay/2022-03-28-spellcheck_dl_en_3_0.md]: you are right. Array() is added.

File: docs/_posts/murat-gunay/2022-03-28-spellcheck_dl_en_3_0.md (hunk 1 above)
Comment (murat-gunay) [reply docs/_posts/murat-gunay/2022-03-28-spellcheck_dl_en_3_0.md]: `new` is added

**Review by josejuanmartinez — APPROVED** [review]

**Pull request merged**